docs/events/wg_workshops/index.md

```diff
@@ -9,4 +9,34 @@ rsecon22/index
 
 ```
 
-The working group has agreed to run quarterly workshops to discuss project progress, latest TRE developments, and actions for the community to take together. This page will record reports and outputs from each of these events.
+The community has agreed to run quarterly workshops to discuss project progress, latest TRE developments, and actions for the community to take together. This page will record reports and outputs from each of these events.
+
+## Background
+
+The community workshops happen four times a year, with 3 sessions fully online, half-day events, and one hybrid (primarily in-person) event once a year.
+The in-person event so far has aligned nicely with RSE Con each year, however as the community builds and develops this may be updated.
```

Comment:
```suggestion
The in-person event so far has aligned nicely with [RSECon](https://rsecon23.society-rse.org/) each year, however as the community builds and develops this may be updated.
```
Either link out, or don't mention RSECon- not all readers will know what RSECon is.